LeoProjectPayment/src/main/java/com/work/LeoProjectPayment/entity/Transaction.java

```diff
@@ -0,0 +1,46 @@
+package com.work.LeoProjectPayment.entity;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Column;
+import lombok.Setter;
+import lombok.Getter;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import java.util.Date;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+@Entity
+public class Transaction {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
```

Comment:
I dont recommend using primitive types, Integer is better. When you put primitives fields cant be nullable, they will be 0 by default.